packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -275,34 +275,32 @@ describe('FLE tests', () => {
         keyVault = client.getKeyVault();
         clientEncryption = client.getClientEncryption();
 
-        // Create necessary data + index keys
-        dataKey1 = keyVault.createKey('local');
-        dataKey2 = keyVault.createKey('local');
-        indexKey = keyVault.createKey('local');
+        // Create necessary data key
+        dataKey = keyVault.createKey('local');
 
         // (re-)create collection -- this needs to be done
         // with the plain mongo client until MONGOCRYPT-435 is done
         coll = client.getDB('${dbname}').encryptiontest;
         Mongo(${uri}).getDB('${dbname}').createCollection('encryptiontest', {
           encryptedFields: {
             fields: [{
-              keyId: indexKey,
+              keyId: dataKey,
```

Comment:
Product decided to drop this feature (using separate data and index keys) pretty late, unfortunately.